pyscada/hmi/static/pyscada/js/pyscada/pyscada_v0-7-0rc14.js

```diff
@@ -2802,7 +2802,7 @@ function createOffset(date) {
                      "November",
                      "December"
                  ],
-                 "firstDay": 1
+                 "firstDay": 0
```

Comment:
this is only the default in most of europe, maybe it ist better to get that info from the locaton settings of the browser?

Reply:
Do you know how and compatible with many browsers ?

Reply:
See : https://stackoverflow.com/questions/53382465/how-can-i-determine-if-week-starts-on-monday-or-sunday-based-on-locale-in-pure-j

Reply:
As the current config is `"daysOfWeek": [
                     "Mo",
                     "Tu",
                     "We",
                     "Th",
                     "Fr",
                     "Sa",
                     "Su",
                 ],`
The current setting is showing `Tuesday` as the first day of the week.

So we can change it to Monday or Sunday till we found how to set it accordingly to the browser settings.

This PR propose Monday.

Reply:
I add a new commit to correct a bug about naming the days.
The day list in the JS datetimepicker config should start on sunday.

I think you can approve this request to fix this bug and then we plan in github `discussion` or `project` to get the browser locale to set the first week day.

What do you think @trombastic ?

Reply:
Okay